common/src/main/java/com/wynntils/commands/FunctionCommand.java

```diff
@@ -34,6 +38,9 @@ public class FunctionCommand extends Command {
                             .map(Function::getName),
                     builder);
 
+    private static final SuggestionProvider<CommandSourceStack> FUNCTION_LIST_TYPES_SUGGESTION_PROVIDER =
```

Comment:
It might be nicer do create an enum and let the provider return `EnumType.values().toLowerCase()`. Or possibly there was even a vanilla type for handling enums, I don't really remember.